test/JDBC/input/PARTITION-vu-prepare.mix

```diff
@@ -575,10 +575,12 @@ SET QUOTED_IDENTIFIER OFF;
 Go
 
 
---------------------------------------------------
---- TO test Permission
---------------------------------------------------
-CREATE DATABASE PartitionDb;
+--------------------------------------------------------------------------
```

Comment:
I do not see any -vu-cleanup for these files.

Reply:
That is not needed here, as we are creating the objects inside user db and which is already being deleted in cleanup scripts.